debug.c

```diff
@@ -94,7 +94,8 @@ void pv_debug_start_ssh()
 	if (db_pid > -1)
 		return;
 
-	if (!pv_config_get_debug_ssh_authorized_keys())
+	if (!pv_config_get_debug_ssh_authorized_keys() ||
+	    !strcmp(pv_config_get_debug_ssh_authorized_keys(), "pv:user-meta"))
```

Comment:
why not just using "default" instead? that way it makes more sense at least from the point of view of pvcontrol

pvcontrol usrmeta save debug.ssh_authorized_keys default

Reply:
ok using __default__ now